### PR TITLE
Added manifest.json + manifest.webapp + lowres.png + address below map

### PR DESCRIPTION
**Pull request by PohSayKeong**

### CODE REVIEW

**Comment by SaptakS** [comment]
Can you please indent the code?

**Comment by PohSayKeong** [comment]
Sry bout that, first time trying to squash my commits, did it, but left a huge mess here...

File: testapi/event/1/event [hunk 1]
```diff
@@ -8,6 +8,7 @@
       "id": 1, 
       "latitude": 52.49814, 
       "location_name": "Station Berlin", 
+	  "address":"STATION 10963 Berlin GmbH Luckenwalder Str. 4–6 10963 Berlin",
```
Comment (championswimmer) [inline testapi/event/1/event]: can we have something like 
```
address : {
  street : 
  locality :
  city : 
  country :
  postal_code: 
}
```

**Comment by PohSayKeong** [comment]
Edited

**Comment by PohSayKeong** [comment]
Fixed, its now postalCode

**Pull request merged**